packages/design-system/src/types/MonthPicker/MonthPicker.d.ts

```diff
@@ -67,14 +67,16 @@ export interface MonthPickerProps {
    * For internationalization purposes, the text for the "Select all"
    * button must be passed in as a prop.
    */
-  selectAllText: string;
+  selectAllText?: string;
```

Comment:
Could we update the PropTypes in `MonthPicker.jsx` too? Its currently listed as required when it should be optional because theres already a default value